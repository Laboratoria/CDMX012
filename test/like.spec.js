/**
 * @jest-environment jsdom
 */
import { like } from '../src/components/like.js';

jest.mock('../src/firebase-imports.js');

describe('Tests of like', () => {
  test('render like', () => {
    const renderlike = like();
    document.body.innerHTML = '<div id="root"></div>';
    const rootDiv = document.getElementById('root');
    rootDiv.appendChild(renderlike);
    expect(rootDiv.innerHTML).toMatchSnapshot();
  });
});
