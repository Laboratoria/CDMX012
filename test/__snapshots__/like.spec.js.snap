// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests of like render like 1`] = `"<div class=\\"like-btn\\"><img src=\\"./assets/like.png\\" alt=\\"like empty button\\"><span class=\\"likeCount\\"></span></div>"`;
